src/index.tsx

```diff
@@ -422,6 +414,32 @@ export class Resizable extends React.PureComponent<ResizableProps, State> {
     return size;
   }
 
+  public componentDidUpdate() {
```

Comment:
@weilao I see. Thanks for your work. however If we improve perf, we should bindEvents on resizeStart and onbind on resizeStop timing.

Reply:
OK

Reply:
Done